### PR TITLE
Add Election Results for the College Council Class of 2027

### PR DESCRIPTION
**Pull request by asteinhart**
## What does this PR do?
Creates a stacked bar chart of the election results using the data found [here](https://docs.google.com/spreadsheets/d/1pFIIRyZJvQ9bWyQfobJmBE184d6OXf_8dJ0x-fN63O8/edit?usp=sharing)

This is an MVP for review just to have something but I may try to make some more edits based on the below notes and/or any feedback you have.

## How was the functionality tested and verified?
- Reviewed local build of the chart and confirmed counts and name spellings matched the spreadsheet above.
- [x] The visual was approved by the story's stakeholders.
  - From Michael M via slack: `The top five candidates were elected” is more in conformity with our style for the sub header, but this is otherwise perfect.` (subtitle was changed)

## Screenshot
v3 

Screenshot 2023-10-25 at 3 29 15 PM

- remove legend
- center titles

v2

Screenshot 2023-10-25 at 2 09 22 PM

v1

Screenshot 2023-10-25 at 12 39 28 PM

## Notes 
1. ~Want to get some more clarity on what `Victorious Candidates` means. Ideally, I would like to highlight these four candidates, possibly by making those colors more saturated and the other candidates a less saturated color~
    - added this
3. Will someone be able to double check that the name spellings are correct on spreadsheet/ chart?
4. I am also going to ask to read the article to see if any other context could help improve the chart

### CODE REVIEW

**Comment by michplunkett** [comment]
Could you post a screenshot of your chart in the description?

File: 2023/sg-election-votes/index.html [hunk 1]
```diff
@@ -0,0 +1,40 @@
+<html lang="en">
+    <head>
+        <title>2023 Student Government Election Results</title>
```
Comment (michplunkett) [inline 2023/sg-election-votes/index.html]: I think this title may work better: `Student Government Election Votes: 2023`. Could you apply it to the HTML page and chart?

File: 2023/sg-election-votes/meta_data.json [hunk 2]
```diff
@@ -0,0 +1,5 @@
+{
+    "_comment": "The description and title fields will be edited to fit your story. This _comment property can can be removed or left as it is, as it will not be included in the fields sent to the server.",
```
Comment (michplunkett) [inline 2023/sg-election-votes/meta_data.json]: You can delete this line.

File: 2023/sg-election-votes/index.html (hunk 1 above)
Comment (asteinhart) [reply 2023/sg-election-votes/index.html]: Went with Student Government Election Results: 2023

**Comment by michplunkett** [comment]
Can you re-run the minifying step?

File: 2023/sg-election-votes/index.html [hunk 3]
```diff
@@ -1,6 +1,6 @@
 <html lang="en">
     <head>
-        <title>Student Government Election Results: 2023</title>
+        <title>College Council Class of 2027 Election Results</title>
```
Comment (michplunkett) [inline 2023/sg-election-votes/index.html]: Thoughts on this one? `Election Results for the College Council Class of 2027`

**Comment by michplunkett** [comment]
For version two of it, you'll need to update your legend as it is no longer correct.

**Comment by asteinhart** [comment]
> For version two of it, you'll need to update your legend as it is no longer correct.

My thought was that the additional legend for losing candidates added more confusion than clarity and the simpler was better but happy to add if you feel otherwise

**Comment by michplunkett** [comment]
If it's between adding an additional legend or having one that doesn't fit, I think it'd be best to remove the legend for the second version. 

I also think it may be worth sticking to v1 in general and then putting a subtitle below saying something to the effect that the top 5 vote totals elected to SG. The multiple colors feels like too much to parse, imo.

File: 2023/sg-election-votes/main.js [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+        type: 'column'
+    },
+    title: {
+        text: 'College Council Class of 2027 Election Results',
+        align: 'left'
+    },
+    subtitle: {
+        text: 'The top five candidates were elected.',
+        align: 'left'
+    },
```
Comment (michplunkett) [inline 2023/sg-election-votes/main.js]: All of our titles and subtitles are centered. 

You could change the subtitle text to `The top five candidates are elected to office.` I think present-tense makes more sense.

**Review by michplunkett — DISMISSED** [review]
Excellent! This is great, @Asteinhart!

**Review by michplunkett — APPROVED** [review]
Great work, merge when you're ready!

**Pull request merged**